internal/db/update.go

```diff
@@ -146,30 +164,43 @@ func UpdatePackedObjects(ctx context.Context, tx pgx.Tx, conn DbConnector, proje
 
 	batch := &pgx.Batch{}
 
-	batch.Queue(`
-		UPDATE dl.objects SET stop_version = $1
-		WHERE project = $2
-		  AND path = $3
-		  AND packed IS true
-		  AND stop_version IS NULL
-	`, version, project, parent)
+	if isStaged {
+		batch.Queue(`
+			INSERT INTO dl.staged_objects (project, start_version, stop_version, path, hash, mode, size, packed)
+			VALUES ($1, NULL, $2, $3, NULL, NULL, NULL, NULL)
```

Comment:
is setting start_version as NULL a pattern used to distinguish the fact that its staged? I see this here and in DeleteObject